guides/common/modules/ref_glossary-of-terms-used-in-project.adoc

```diff
@@ -114,11 +173,13 @@ ERB:: Embedded Ruby (ERB) is a template syntax used in provisioning and job temp
 Errata:: Updated RPM packages containing security fixes, bug fixes, and enhancements.
 In relationship to a host, erratum is *applicable* if it updates a package installed on the host and *installable* if it is present in the host's content view (which means it is accessible for installation on the host).
 
-[[External_node_classifier]]
-External node classifier:: A construct that provides additional data for a server to use when configuring hosts.
-{ProjectName} acts as an External Node Classifier to Puppet servers in a {Project} deployment.
+[[External_node_classifier_ENC]]
+External node classifier (ENC)::
+A construct that provides additional data for a server to use when configuring hosts.
+When Puppet obtains information about nodes from an external source instead of its own database, the external source is called External node classifier.
+If the Puppet plugin is installed, {ProjectName} can act as an External node classifier to Puppet servers in a {Project} deployment.
 +
-The External Node Classifier will be removed in a future {Project} version.
+The External node classifier will be removed in a future {Project} version.
```

Comment:
Is is External still capitalized?

Also, I think we could even drop this whole line now because I don't think we will drop it.

Reply:
Dropped this line.